MonoMod.UnitTest/MonoMod.UnitTest.csproj

```diff
@@ -14,7 +14,7 @@
     <OutputType>Library</OutputType>
     <SignAssembly>skip</SignAssembly>
     <!-- FIXME: RE-ENABLE TESTS WITH TIERED COMPILATION -->
-    <TieredCompilation>false</TieredCompilation>
+    <!--<TieredCompilation>false</TieredCompilation>-->
```

Comment:
The FIXME here seems to no longer be relevant as we are now testing with tiered compilation.